src/test/scala/cromwell/ThreeStepActorSpec.scala

```diff
@@ -130,17 +130,19 @@ class ThreeStepActorSpec extends CromwellSpec(ActorSystem("ThreeStepActorSpec",
   import ThreeStepActorSpec._
 
   "A three step workflow" should {
-      "best get to (three) steppin'" in {
+    "best get to (three) steppin'" in {
       val fsm = buildWorkflowActorFsm()
       assert(fsm.stateName == WorkflowSubmitted)
-      fsm ! Start
       within(TestExecutionTimeout) {
-        awaitCond(fsm.stateName == WorkflowRunning)
-        awaitCond(fsm.stateName == WorkflowSucceeded)
-        val outputs = Await.result(fsm.ask(GetOutputs)(ActorTimeout).mapTo[WorkflowOutputs], 5 seconds)
-        val Seq(cgrepCount, wcCount) = getCounts(outputs, "three_step.cgrep.count", "three_step.wc.count")
-        cgrepCount shouldEqual 3
-        wcCount shouldEqual 6
+      startingCallsFilter("ps").intercept {
```

Comment:
thanks, I had purposefully committed broken code and I was going to come back to it last night but then a rebase got me and I never got to it :-/